GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -289,11 +369,22 @@ protected async Task ReloadNodesAsync(Func<CancellationToken, Func<RefsFilter, I
 
                 await treeView.SwitchToMainThreadAsync(token);
 
+                // remember multi-selected nodes
+                var multiSelected = this.GetMultiSelection().Select(node => node.GetHashCode()).ToArray();
```

Comment:
Use explicit types unless the type is obvious from the context (which isn't the case here).